content/rule_book/material_and_questions.md

```diff
@@ -0,0 +1,275 @@
+
+As an example of referencing the determining word or phrase, consider 2 Peter 2:8:
+
+> For that righteous man, living among them day after day, was tormented in his righteous soul by the lawless deeds he saw and heard.
+
+A CR question could be written: "What deeds?" The answer would be: "Lawless." If the quizzer provided the question, "Lawless what?" the quizzer would be incorrect because the quizzer's question and the written question are not similar.
+
+##### Commentary
+
+If 2 identical phrases exist in the same chapter, but 1 occurrence grammatically calls for use of the "who" inserted interrogative word, and the other occurrence grammatically calls for the use of the "what" inserted interrogative word, that difference in the grammatically dictated interrogative word does not change the fact that the phrase from the material (that excludes the inserted interrogative word) occurs more than twice in the same chapter, necessitating a CVR question.
+
+The above does not mean that if the quizzer needs to provide a reference question that is a CVR, and their choice of interrogative word changes the question type to MACVR, that the quizzer has given a valid CVR. In this case, the validity changes based on the interrogative word because of how the required answer changes.
+
+##### Chapter Verse Reference (CVR)
+
+- Must have a single answer
+- The question must contain a word or phrase that exists in more than 1 verse in the same chapter
+- Context is the single verse
+
+The question text should be in the form:
+
+> According to [book name], chapter [chapter number], verse [verse number], [remainder of question text]
+
+##### Chapter Reference (CR)
+
+- Must have a single answer
+- Must contain a word or phrase that exists more than once in the material and only once in the chapter
+
+The question text should be in the form:
+
+> According to [book name], chapter [chapter number], [remainder of question text]
+
+##### Multiple Answer Chapter Verse Reference (MACVR)
+
+- Must meet all the requirements of a CVR except the single answer requirement
+- Must also meet all the requirements of a MA except the unique word or phrase requirement
+
+##### Multiple Answer Chapter Reference (MACR)
+
+- Must meet all the requirements of a CR except the single answer requirement
+- Must also meet all the requirements of a MA except the unique word or phrase requirement
+
+#### Quote/Finish
+
+Quote and finish questions must be quoted word-perfect. Quizzers may go back and correct their answers if done in the question answer duration period. The quizzer will be required to make 1 full, correct rotation; however, the quizzer can be called correct without 1 full, correct rotation if they have made a mistake only in 1 area of the verse and correct the mistake, making it abundantly clear to the quizmaster and answer judge that they know where their mistake was and what the corrected words are.
```

Comment:
**Received via an email:** *was it intentional to not have description of what a ftv/qt should be? that opens up to any verse at international being possibilities for ftv or qt? not sure I love that.*

Reply:
We moved the "strong enough to stand on its own" and "spiritually significant" concepts for Quote/Finish questions to the best practices document since we couldn't figure out a way to word these 2 things objectively without violating functional equivalence. One of our goals was to have the core rule book be completely objective, with every aspect being challenge-able or protest-able.

Here's a link to the best practices document:

https://github.com/gryphonshafer/Quizzing-Rule-Book/blob/b1010029558bfdb36d3ba1285f94e6ef27b43c8c/content/best_practices.md

The "strong enough to stand on its own" and "spiritually significant" concepts from the 2018 rule book are so subjective as to be easily ignore-able by anybody who wants to ignore them. There are some who believe "Internationals quizzers should know the whole material and F/Q question types should test them on the whole material." It's an opinion not supported anywhere in any rule book, but if someone has that opinion and also has power over the question set for a meet, that opinion gets enacted. And there are some districts that ignore completely the "spiritually significant" language in the 2018 rule book.

In the next version of the rule book, we're going to be looking at ways coaches can raise protests about the meet, not just rulings on challenges. This could provide a mechanism for coaches to push-back on poor question sets.